start.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh -e
+
+# TODO Consider moving all the TODO to proper issues
```

Comment:
Lots of great ideas here! Please feel free to move these to issues and remove them from this file. Although, a few of them can stay, as the TODO item might make more sense when it is inline with the code. To that end I'll follow your judgement.

Reply:
```suggestion
```